src/python/pants/option/global_options.py

```diff
@@ -438,17 +438,16 @@ def from_options(
         prior_result: AuthPluginResult | None = None,
         remote_auth_plugin_func: Callable | None = None,
     ) -> tuple[DynamicRemoteOptions, AuthPluginResult | None]:
-        bootstrap_options = full_options.bootstrap_option_values()
-        assert bootstrap_options is not None
-        execution = cast(bool, bootstrap_options.remote_execution)
-        cache_read = cast(bool, bootstrap_options.remote_cache_read)
-        cache_write = cast(bool, bootstrap_options.remote_cache_write)
+        global_options = full_options.for_global_scope()
```

Comment:
The use of bootstrap options was always gratuitous here. We have a fully formed Options instance, so these are just regular global options now.